### DISCUSSION TITLE
What is the equivalent of D in neovim?

### DISCUSSION THREAD

**Discussion by OpenFoam-User** [helix-editor/helix#4429] [Q&A] [answered]
Hello

In insert mode one can delete from the cursor to the end of line using: <kbd>Ctrl</kbd> <kbd>k</kbd>
However, I can't find the equivalent command to do the same thing in _normal_ mode (in neovim one should press <kbd>D</kbd>).

Your help is very much appreciated.

**Comment by geometryolife** [comment]
Currently in normal mode, `D` is not bound to any function.

**Reply by OpenFoam-User** [reply]
Thanks for the reply. if it is not bound to any function ,how can I bind it to what I am asking for?

**Reply by David-Else** [reply]
`config.toml`
```
[keys.normal]
D = ["extend_to_line_end", "delete_selection"]
```
and here are my other vim essentials as a bonus :)
```
Z = { Z = ":wq", Q = ":q!" }                                         # save and quit, quit without saving
G = "goto_file_end"
C = ["collapse_selection", "extend_to_line_end", "change_selection"]
```

**Comment by lisiur** [comment]
vgld

**Comment by Omnikar** [comment, marked as answer]
`t<ret>d` is the easiest way to do it without changing keybinds.

**Reply by alexsunxl** [reply]
If the target deleted line is the last line of buffer, it can't be work...

**Reply by oihv** [reply]
> If the target deleted line is the last line of buffer, it can't be work...

I tested it just now, it does work.

**Comment by Ordoviz** [comment]
In your config.toml:
```toml
[keys.normal]
D = ["extend_to_line_end", "delete_selection"]
```
